src/qibocal/auto/serialize.py

```diff
@@ -22,6 +22,11 @@ def serialize(raw: dict):
 def load(key):
     """Evaluate key converting string of lists to tuples."""
     raw_load = json.loads(key)
-    if isinstance(raw_load, list):
-        return tuple(raw_load)
-    return raw_load
+
+    def to_tuple(raw):
+        if isinstance(raw, list):
+            return tuple([to_tuple(item) for item in raw])
+        else:
+            return raw
```

Comment:
```suggestion
        if isinstance(raw, list):
            return tuple([to_tuple(item) for item in raw])
        return raw
```
if the former branch is returning, you do not need an `else:` branch (this is flagged by Pylint as well).